lib/jsonapi/utils/underscore.ex

```diff
@@ -42,9 +42,13 @@ defmodule JSONAPI.Utils.Underscore do
       "_top__posts_"
   """
   def underscore(value) when is_atom(value) do
```

Comment:
These method names seem confusing to me :). Really it is `dasherize`

Reply:
:) yes I agree, they would make more sense to me if switched